File: VS Encrytions cloud/2023/other/PY_Flask/3.py
# -*- coding: utf-8 -*-
# @Author: Dastan_Alam
# @Date:   29-06-2023 01:17:08 PM       13:17:08
# @Last Modified by:   Dastan_Alam
# @Last Modified time: 29-06-2023 01:17:15 PM       13:17:15


from flask import Flask, render_template, request
import csv

import os

app = Flask(__name__, template_folder="temp")
app.config['UPLOAD_FOLDER'] = '2023/PY_Flask/upload'

@app.route('/')
def index():
    return render_template('index.html')

@app.route('/upload', methods=['POST'])
def upload():
    file = request.files['file']
    if file:
        csv_data = file.read().decode('utf-8')  # Read the uploaded CSV data
        csv_rows = csv_data.split('\n')  # Split the CSV data into rows
        num_rows = len(csv_rows) - 1  # Subtract 1 to exclude the header row

        return f'The number of rows in the CSV file is: {num_rows}'
    else:
        return 'No file selected!'

if __name__ == '__main__':
    app.run(debug=True)

File: VS Encrytions cloud/2022/c c++ Encrypt/c++ f/3.cpp
/**
 * @Author: DASTAN_ALAM
 * @Date:   13-05-2022  Fri 12:13:22 PM  12:13:22
 * @Last Modified by:   DASTAN_ALAM
 * @Last Modified time: 13-05-2022  Fri 12:16:35 PM  12:16:35
 */
#include <iostream>
using namespace std;
// no argument but return valuee  
// without argument and with return valuee 
int value();
int main()
{
    int a;
    a=value();
    cout<<a;
    return 0;
}
int value()
{
    int b,c,d;
    b=5;c=5;
    d=b+c;
    return d;
}
